framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -204,7 +204,8 @@ def hide_launch_and_setup_pse_button(self):
 
     @property
     def include_branch_button(self):
-        if self.is_launch_fragment and self.is_simulation_copy and not self.is_pse_launch:
+        is_branch = common.get_from_session(common.KEY_BRANCH)
+        if is_branch and self.is_launch_fragment and self.is_simulation_copy and not self.is_pse_launch:
```

Comment:
I think this condition can be simplified now, as now we want to show the branch button only when `is_branch` and `is_launch_fragment`  are true, we don't want it on the 'copy' page anymore